### PR TITLE
pchain tx parser fix

### PR DESCRIPTION
**Pull request by abi87**
I believe we are skipping lockable inputs

### CODE REVIEW

File: mapper/pchain/tx_ops.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package pchain
+
+import "github.com/coinbase/rosetta-sdk-go/types"
+
+// txOps collects all balance-changing information within a transaction
+type txOps struct {
```
Comment (abi87) [inline mapper/pchain/tx_ops.go:6]: just moved to a different file

File: mapper/pchain/tx_parser.go [hunk 2]
```diff
@@ -384,16 +385,22 @@ func (t *TxParser) insToOperations(
 			Type: metaType,
 		}
 
-		if transferInput, ok := in.In.(*secp256k1fx.TransferInput); ok {
-			metadata.SigIndices = transferInput.SigIndices
+		input := in.In
+		if stakeableIn, ok := input.(*stakeable.LockIn); ok {
```
Comment (abi87) [inline mapper/pchain/tx_parser.go:389]: we do something similar with outputs, not sure why we would process inputs differently

File: service/backend/pchain/block.go [hunk 3]
```diff
@@ -68,12 +68,12 @@ func (b *Backend) Block(ctx context.Context, request *types.BlockRequest) (*type
 		}
 		blockIndex = int64(block.Height)
 
-		dependencyTxs, err := b.fetchDependencyTxs(ctx, block.Txs)
+		blkDeps, err := b.fetchBlkDependencies(ctx, block.Txs)
```
Comment (abi87) [inline service/backend/pchain/block.go:71]: some leftover renaming

File: mapper/pchain/tx_parser_test.go [hunk 4]
```diff
@@ -28,10 +28,9 @@ var (
 func TestMapInOperation(t *testing.T) {
 	_, addValidatorTx, inputAccounts := buildValidatorTx()
 
-	assert.Equal(t, 1, len(addValidatorTx.Ins))
+	assert.Equal(t, 2, len(addValidatorTx.Ins))
```
Comment (abi87) [inline mapper/pchain/tx_parser_test.go:31]: added a locked time input

File: mapper/pchain/tx_parser.go (hunk 2 above)
Comment (rajranjan0608) [reply mapper/pchain/tx_parser.go:389]: lgtm. @bilgehansahin this should resolve the missing locked-ins bug.

**Pull request merged**